src/platform/silabs/efr32/OTAImageProcessorImpl.cpp

```diff
@@ -318,6 +333,7 @@ CHIP_ERROR OTAImageProcessorImpl::SetBlock(ByteSpan & block)
 {
     if ((block.data() == nullptr) || block.empty())
     {
+        ChipLogError(SoftwareUpdate, "%s: block is empty", __func__);
```

Comment:
This isn't a "error", this condition returns `CHIP_NO_ERROR;`

don't log it as a `ChipLogError` you should probably not log it at all.